pep-0011.txt

```diff
@@ -158,13 +158,15 @@ Microsoft has established a policy called product support lifecycle
 [1]_. Each product's lifecycle has a mainstream support phase, where
 the product is generally commercially available, and an extended
 support phase, where paid support is still available, and certain bug
-fixes are released (in particular security fixes).
+fixes are released. Sometimes this is then followed by a third phase
+called "ESU", for "extended security updates".
```

Comment:
I would (and did, in https://github.com/python/cpython/issues/76773#issuecomment-1093773623 ) describe ESU not as a "phase" but rather a specialized, paid program for high-volume enterprise customers, and be more explicit here (as well as below) in mentioning ESU specifically in the context of making clear the distinction with the Extended Support phase and CPython support, rather than making it part of "this lifecycle" that CPython is stated to follow below.

So, how about (incorporating @hugovk 's comments as well):

```suggestion
fixes are released. This is distinct from `ESU (Extended Security Updates)
<https://learn.microsoft.com/en-us/lifecycle/faq/extended-security-updates>`_,
a specialized, paid program available to high-volume enterprise customers
as a "last resort" option after the end of extended support.
```